portfolio/src/main/webapp/index.html

```diff
@@ -9,11 +9,54 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+ 
+      <h3>Background</h3>
+      <p>Hello, my name is Kevin Alfaro and this is my portfolio!</p>
+      <p>I currently am attending the University of Washington. I am a junior 
```

Comment:
nit: if the text spans multiple lines, I prefer to have a line break so it's easier to read
```
<p>
  Your text content
  that spans multiple lines
</p>
```
I see that some of the text below follows this. I would say choose one style and be consistent

Reply:
Done!